src/parser/jsoncons/json_parser.hpp.orig

```diff
@@ -0,0 +1,2157 @@
+// Copyright 2015 Daniel Parker
```

Comment:
I remove this file, and simplifyDriver compiles with no problems. I think,
the core json functionality that we are using, does not depend on this file.

On Thu, Jun 23, 2016 at 8:53 PM, Michelle Strout <notifications@github.com>
wrote:

> In src/parser/jsoncons/json_parser.hpp.orig
> <https://github.com/CompOpt4Apps/IEGenLib/pull/38#discussion_r68266567>:
>
> > @@ -0,0 +1,2157 @@
> > +// Copyright 2015 Daniel Parker
>
> This file has numerous conflicts in it. If we aren't using it, then let's
> remove it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CompOpt4Apps/IEGenLib/pull/38/files/873a894609e6a1eb8305ba539144baac69001790#r68266567>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ANGVlyW90CHeqBdyU76QIn9KDxQEZzw-ks5qOrMXgaJpZM4I89jG>
> .
>